UnitsNet.Benchmark/Operators/Additions/SumOfTemperatureDeltasWithSameUnitsBenchmarks.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Licensed under MIT No Attribution, see LICENSE file at the root.
+// Copyright 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com). Maintained at https://github.com/angularsen/UnitsNet.
+
+using System.Linq;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Jobs;
+using UnitsNet.Units;
+
+namespace UnitsNet.Benchmark.Operators.Additions;
+
+[MemoryDiagnoser]
+[SimpleJob(RuntimeMoniker.Net48)]
+[SimpleJob(RuntimeMoniker.Net80)]
+public class SumOfTemperatureDeltasWithSameUnitsBenchmarks
+{
+    private static readonly double Value = 1.23;
+
+    private TemperatureDelta[] _quantities;
+
+    [Params(10, 1000)]
+    public int NbOperations { get; set; }
+
+    [Params(TemperatureDeltaUnit.Kelvin, TemperatureDeltaUnit.DegreeCelsius, TemperatureDeltaUnit.DegreeFahrenheit)]
+    public TemperatureDeltaUnit Unit { get; set; }
+
+    [GlobalSetup]
+    public void PrepareQuantities()
+    {
+        _quantities = Enumerable.Range(0, NbOperations).Select(_ => TemperatureDelta.From(Value, Unit)).ToArray();
+    }
+
+    [Benchmark(Baseline = true)]
+    public TemperatureDelta SumOfDeltas()
+    {
+#if NET
+        return UnitsNet.GenericMath.GenericMathExtensions.Sum(_quantities);
```

Comment:
I think using `GenericMathExtension` for net50+ and regular addition for net48 is an apples to oranges comparison, ref the `#if NET` condition.

I see that all the "Sum" tests involve `GenericMathExtensions`, but I think we should have dedicated tests for that class vs performance tests of the addition operators alone, to get meaningful results.

Reply:
In my "Fraction" based implementation I've got the same extension prototype applicable to both types- so I'm comparing NET Framework v6 to NET Framework v6 with the new extension.
By itself the test only shows how much faster (ratio) the NET is compared to NET Framework when adding a bunch of quantities (also- the scaling doesn't linear with the fractions).